app/controllers/school_classes_controller.rb

```diff
@@ -0,0 +1,31 @@
+class SchoolClassesController < ApplicationController
+  def create
+    @school_class = SchoolClass.create(classes_params)
+    redirect_to school_class_path(@school_class)
+  end
```

Comment:
Since you're not using the `@school_class` instance variable in a view, it could be a regular variable.